build_scripts/90-image-info.sh

```diff
@@ -30,9 +29,9 @@ CODE_NAME="Achillobator Giganticus"
 sed -i -f - /usr/lib/os-release <<EOF
 s/^NAME=.*/NAME=\"${IMAGE_PRETTY_NAME}\"/
 s|^VERSION_CODENAME=.*|VERSION_CODENAME=\"${CODE_NAME}\"|
-s/^ID="centos"/ID=${IMAGE_PRETTY_NAME,}\nID_LIKE=\"${IMAGE_LIKE}\"/
+s/^ID="centos"/ID=bluefin_lts/
```

Comment:
I just realized there's really no need to match on `"centos"` at all.

```suggestion
s/^ID=.*/ID=bluefin_lts/
```

Reply:
Oh man I wonder who would like that!

(just kidding this looks great)